amy/dashboard/tests/test_get_involved_views.py

```diff
@@ -21,14 +20,32 @@ def setUp(self):
                 "involvement_required": True,
             },
         )
-        self.github_contribution, _ = Involvement.objects.get_or_create(
+        self.involvement, _ = Involvement.objects.get_or_create(
             name="GitHub Contribution",
             defaults={
                 "display_name": "GitHub Contribution",
                 "url_required": True,
                 "date_required": True,
             },
         )
+
+        # set up and log in as a trainee
+        self.user = Person(
+            username="trainee_alice",
+            personal="Alice",
+            family="Trainee",
+            email="alice_trainee@example.com",
+            is_active=True,
+        )
+        self.user.set_password("password")
```

Comment:
Alternatively you can use `Person.objects.create_user()` (method of `PersonManager`) which accepts password.